cellpack/autopack/__init__.py

```diff
@@ -261,8 +250,49 @@ def updateReplacePath(newPaths):
         REPLACE_PATH[w[0]] = w[1]
 
 
+def download_file_from_s3(s3_uri, local_file_path):

```

Comment:
it seems like this function does the same thing as `def download_file()` and isn't being used. we can keep one of them and remove the other

Reply:
Good point!